python/qpid_dispatch_internal/tools/display.py

```diff
@@ -138,6 +143,33 @@ def num(self, value):
     value /= 1000
     return self.numCell(value, 'g')
 
+def PlainNum(value):
```

Comment:
Also, why "PlainNum" and not "Integer", since %d requires an integer argument?

Reply:
I went with PlainNum since no formatting was done, do you want me to call it PlainInteger or just Integer ?

Reply:
I am keeping it as PlainNum since you were ok with it